.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,9 +3,7 @@ name: Bug report
 about: Found a bug in Fabric? Please let us know.
 
 ---
-<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨
-
-We’re here to help, but we need a few things from you! 
+<!--  
 
 Please help us reduce duplicates by doing the following steps before logging an issue:
```

Comment:
I'm not sure if this section is necessary either. I don't mind asking people to check for duplicates, but long URLs that you have to copy and paste feel like a lot of noise to me.

Reply:
Before you log the issue, have we already answered it? Search here!